src/revlanguage/workspace/RbRegister_Move.cpp

```diff
@@ -25,7 +25,7 @@
 
 #include <sstream>
 #include <vector>
-#include <cstdio>
```

Comment:
This should change back to <cstdio> - probably a merge issue.

Reply:
Done.